dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostFactoryImpl.java

```diff
@@ -791,6 +791,17 @@ protected Optional<List<Host>> search(final String siteNameFilter, final String
             hostList.addAll(APILocator.getPermissionAPI().filterCollection(siteList, PermissionAPI
                     .PERMISSION_READ, respectFrontendRoles, user));
             hostList = hostList.stream().limit(limit).collect(Collectors.toList());
+
+            //We want to include the system host in the list of hosts
```

Comment:
This change looks good but still might have undesired side effect returning System Host here